src/restLink.ts

```diff
@@ -1393,12 +1422,28 @@ export class RestLink extends ApolloLink {
             resolverOptions,
           )
             .then(data => {
-              setContext({
-                restResponses: (context.restResponses || []).concat(
+              /** Composability Modification:
+               * not sure why the code is caching setContext but we want to make sure
+               * we're setting context the the right point in the chain and using a cached
+               * setContext can break this
+               */
```

Comment:
I don't believe the caching of `setContext` was intentionally thought through. I think that was just an accident of implementation, no objection to this change -- though we're instead caching `operation`.

```suggestion
```